packages/ember-runtime/lib/mixins/enumerable.js

```diff
@@ -811,7 +820,9 @@ var Enumerable = Mixin.create({
     @public
   */
   without(value) {
```

Comment:
I think we have a problem line 832 during `k !== value`. If the feature is enabled, `without` become inconsistent because `NaN !== NaN` is `true`